src/org/omegat/help/Help.java

```diff
@@ -69,13 +69,17 @@ private Help() {
 
     /**
      * URL for the online manual.
+     * TODO implement a "manual-dev" and "javadoc-dev"
+     * since the "latest" suffix is not used anymore
      */
     public static final String ONLINE_HELP_URL = OStrings.IS_BETA
-            ? "https://omegat.sourceforge.io/manual-latest/"
+            //? "https://omegat.sourceforge.io/manual-latest/"
+            ? "https://omegat.sourceforge.io/manual-standard/"
             : "https://omegat.sourceforge.io/manual-standard/";
```

Comment:
Why not remove redundant conditional operator?

It can be `public static final String ONLINE_HELP_URL = "https://omegat.sourceforge.io/manual-standard/";`